### ISSUE TITLE
Draggable Taskbar Bug

### ISSUE THREAD

**Issue by Aegim** [MicrosoftDocs/windows-uwp#4042]
By using Screen Ruler, you may accidentally drag your taskbar to an abnormal position. You can also drag it back, so restarting is unnecessary. 

To Recreate: 

1) Press the Screen Ruler activation shortcut

2) Drag taskbar

3) See Issue

---
#### Document Details

⚠ *Do not edit this section. It is required for docs.microsoft.com ➟ GitHub issue linking.*

* ID: 51f06fdf-c629-78d6-7740-ad6162cda4b8
* Version Independent ID: 51f06fdf-c629-78d6-7740-ad6162cda4b8
* Content: [PowerToys Screen ruler utility for Windows](https://docs.microsoft.com/en-us/windows/powertoys/screen-ruler)
* Content Source: [hub/powertoys/screen-ruler.md](https://github.com/MicrosoftDocs/windows-dev-docs/blob/docs/hub/powertoys/screen-ruler.md)
* Product: **dev-environment**
* Technology: **powertoys**
* GitHub Login: @mattwojo
* Microsoft Alias: **mattwoj**

**Comment by Aegim** [comment]
Apologies, as I have never had this issue before. I only noticed this after installing the new update, v0.62.0.

**Comment by crutkas** [comment]
Can you file this over on Powertoys?  We need more info like OS version. 

https://github.com/microsoft/PowerToys/issues/new?assignees=&labels=Issue-Bug%2CNeeds-Triage&template=bug_report.yml

**Comment by Aegim** [comment]
@crutkas 

To my chagrin, I only noticed this issue when using Screen Ruler. I had failed to realize said scenario could be recreated at any moment; so I apologize, for I am afraid that my so-called "issue" has wasted others' time with an oversight of mine. Before I close what had been brought forth by my failure to notice a feature of Win10 for years, I will test some things. My ignorance caused unnecessary allocation of manpower at the expense of distracting those who are doing their best to fix legitimate issues. To anyone I inconvenienced, I sincerely apologize. 

Best regards to the team, and have a nice day.
